flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/data_mapper_model/source/function_call1.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/http;
+
+type UserInfo record {|
+   string username;
+   string password;
+|};
+
+type Student record {|
+   string username;
+   string password;
+|};
+
+const string CONST = "CONST";
+
+service OASServiceType on new http:Listener(9090) {
+
+	resource function get pet() returns int|http:NotFound {
+        do {
+            UserInfo userInfo = {username: "un", password: "pw"};
+            // Student student = getStudent(userInfo);
```

Comment:
Why is this commented?

Reply:
This is the functions call statement that is going to add by the test